index.html

```diff
@@ -77,6 +77,230 @@ <h3>LET'S CONNECT</h3>
         </li>      
        </ul>
     </section>
+    <section class="portfolio" id="portfolio">
+      <nav class="portfolio-border">
+        <img
+          class="portfolio-image"
+          alt="portfolio project"
+          src="Snapshoot_Portfolio1.png"
+        />
+        <div class="project-section">
+          <h3 class="project-name">Tonic</h3>
+          <nav class="project-info">
+            <h4 class="project-client">CANOPY</h4>
+            <img src="Counter.png" class="counter" alt="counter" />
+            <span class="programming-language">Back End Dev</span>
+            <img src="Counter.png" class="counter" alt="counter" />
+            <span class="year">2015</span>
+          </nav>
+
+          <p class="project-details">
+            A daily selection of privately personalized reads; no accounts or <br />
+            sign-ups required. 
+          </p>  
+          <ul class="languages">
+            <li class="dev-tool">html</li>
+            <li class="dev-tool">css</li>
+            <li class="dev-tool">javascript</li>
+          </ul>
+          <button class="btn">See Project</button>
+        </div>
+      </nav>
```

Comment:
- [ ] I see that you are using `nav` tag here and it is semantically not right. Please consider using a more appropriate `ul` as the project container and `li` for each project. 
- [ ] Also, you have a `button` tag without a type attribute declared. Please always ensure that every button has this attribute set.  In this case, it should be `type="button"`. [Read more here](https://medium.com/@clairecodes/why-its-important-to-give-your-html-button-a-type-193b4226fa0a#:~:text=When%20a%20button%20doesn't,maintain%20and%20understand%20for%20others.) on why it is important.

Reply:
Good job following the previous reviewer's suggestions (BUTTON).